package.json

```diff
@@ -42,14 +42,13 @@
   },
   "peerDependencies": {
     "@apollo/composition": "^2.0.4",
-    "@apollo/federation": "^0.36.1",
     "@apollo/gateway": "^2.0.4",
     "graphql": "^16.5.0"
   },
   "devDependencies": {
-    "@apollo/composition": "^2.0.4",
-    "@apollo/federation": "^0.36.1",
-    "@apollo/gateway": "^2.0.4",
+    "@apollo/composition": "^2.5.1",
+    "@apollo/gateway": "^2.5.1",
+    "@apollo/subgraph": "^2.5.1",
```

Comment:
shouldn't these libs be `dependencies` ?
Some of the items exported by the lib use these dependencies.
- Since apolloServer doesn't ship these dependencies, they can't be just peerDependencies also.